WooCommerce/Classes/View Modifiers/View+AutofocusTextEditor.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+/// Autofocus for `TextField` and `TextEditor` in iOS 15 and later
+///
+struct AutofocusTextField: ViewModifier {
```

Comment:
```suggestion
struct AutofocusTextModifier: ViewModifier {
```

May express the intention better?

Reply:
Yes, this is better.